lib/cluster/election.c

```diff
@@ -628,6 +628,7 @@ election_count_vote(election_t *e, xmlNode *message, bool can_win)
 
     } else if (done == FALSE && we_lose == FALSE) {
         int peers = 1 + g_hash_table_size(crm_peer_cache);
+        bool wrote_blackbox = FALSE;
```

Comment:
wouldn't that need to be static to have an effect?

Reply:
whoops, yes